cmd/start.go

```diff
@@ -72,20 +71,15 @@ var (
 
```

Comment:
replace fmt.Println(err) in line 64 with
```suggestion
logger.Error(err.Error())
```